src/Builder/FormContractorInterface.php

```diff
@@ -31,14 +31,16 @@ public function __construct(FormFactoryInterface $formFactory);
      *
      * @return FormBuilderInterface
      */
-    public function getFormBuilder($name, array $options = []);
+    public function getFormBuilder($name, array $formOptions = []);
 
     /**
+     * NEXT_MAJOR: Change signature to add the third parameter $formOptions.
+     *
      * Should provide Symfony form options.
      *
      * @param string|null $type
      *
      * @return array
      */
-    public function getDefaultOptions($type, FieldDescriptionInterface $fieldDescription);
+    public function getDefaultOptions($type, FieldDescriptionInterface $fieldDescription/*, array $formOptions = []*/);
```

Comment:
Can you add `@phpstan-param` information for this new parameter

Reply:
I tried, and phpstan is giving an error.